app/code/core/Mage/Api/etc/config.xml

```diff
@@ -111,6 +111,18 @@
             </modules>
         </translate>
     </adminhtml>
+    <crontab>
+        <jobs>
+            <api_session_cleanup>
+                <schedule>
+                    <cron_expr>0 0 * * *</cron_expr>
```

Comment:
should this be configurable like some other cron running time?

Reply:
Can be configurable but unnecessary, it just needs to run once a day.  The first run may take longer time, subsequently it should be rather fast.

Reply:
cause at midnight many things start and it could be a tricky time for some websites

Reply:
OK, I can make it configurable, wait for it.

Reply:
I don't think it needs to be configurable, I like to just pick a "random" time that is off-peak to avoid always using midnight exactly when there is not a reason to run at a specific time (anything but a time-sensitive report).

```suggestion
                    <cron_expr>0 35 * * *</cron_expr>
```